src/Tracking/DisasterTrackerApp.WebApi/Controllers/WarningController.cs

```diff
@@ -33,4 +33,21 @@ await _warningService.GetWarningEvents(warningRequest, cancellationToken)
             .ToTask(cancellationToken);
        
     }
+    [HttpGet("/receive-statisticwarnings")]
```

Comment:
just a thought.  maybe it is possible to use merge or concatenation of observables to use one end-point /receive-warnings?